js/src/components/paid-ads/asset-group/asset-group.js

```diff
@@ -108,14 +120,22 @@ export default function AssetGroup( { campaign } ) {
 		recordGlaEvent( 'gla_submit_campaign_button_click', eventProps );
 	}
 
+	const recordActionedCampaign = () => {
+		if ( hasRaiseBudgetRecommendation() ) {
+			upsertActionedCampaign( campaign.id );
+		}
+	};
+
 	const handleSkipClick = ( event ) => {
 		handleSubmit( event );
+		recordActionedCampaign();
```

Comment:
I don't think we want to dismiss the recommendation if the campaign editor is closed without changing the budget.

Reply:
@joemcgill Actually the skip button in this case is for adding assets to the campaign (step 2 of the campaign editing screen). On the first screen of the campaign editing, the user can still choose the budget amount and skip the second step and the campaign will be saved.